src/rpc/misc.cpp

```diff
@@ -103,6 +103,7 @@ static RPCHelpMan createmultisig()
                         {RPCResult::Type::STR, "address", "The value of the new multisig address."},
                         {RPCResult::Type::STR_HEX, "redeemScript", "The string value of the hex-encoded redemption script."},
                         {RPCResult::Type::STR, "descriptor", "The descriptor for this multisig"},
+                        {RPCResult::Type::STR, "warnings", "Any warnings resulting from the creation of this multisig"},
```

Comment:
I think it would make sense to make warnings an array of strings.

Reply:
Sure, although this is in-line with how other RPCs return warnings.

Reply:
Warnings array is also preferable to me. There are a few other RPCs which return warnings array, and I think we should move to doing that more.

Reply:
Done.